### ISSUE TITLE
Stamen maps not working

### ISSUE THREAD

**Issue by giobiondani** [BruTile/BruTile#233]
I am using BruTile through Mapsui and I noticed that the StamenTerrain KnownTileSource has stopped working - the map is empty.

I wonder if this has something to do with the Stamen maps moving to Stadia and requiring a Stadia API key.

**Comment by giobiondani** [comment]
I was able to make it work by replacing the URL formatter in the KnownTileSources with this:

`"https://tiles.stadiamaps.com/tiles/stamen_terrain/{z}/{x}/{y}.png?api_key=" + StadiaAPIKey`

**Comment by pauldendulk** [comment]
You added your personal api key? Perhaps I should request an api key for the demo.

**Comment by giobiondani** [comment]
yes, in my app I created this method that is called when I need this map:
```
        public static HttpTileSource StamenTerrain() {
            return new HttpTileSource(
                new GlobalSphericalMercator(4, 19),
                "https://tiles.stadiamaps.com/tiles/stamen_terrain/{z}/{x}/{y}.png?api_key=" + StadiaAPIKey,
                null, apiKey: null, name: null,
                persistentCache: null, tileFetcher: null,
                attribution: GetStadiaAttribution(), userAgent: null
            );
        }
```
the StadiaAPIKey variable refers to my own key, which is configured elsewhere in the app.
This works fine.
I tried to use the "apiKey" parameter for the API key but it wouldn't work that way, that's why I included it in the URL Formatter.
